.github/workflows/ruby.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
+name: Ruby
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        os: ["ubuntu-latest"]
+        ruby: ["2.3", "2.4", "2.5", "2.6"]
```

Comment:
Any chance 2.7 can be added? I don't think they have a ruby-head option but at least would help to have it run against 2.7 since it includes important deprecations.

Reply:
Sure thing!